lib/src/uptech_growthbook_wrapper.dart

```diff
@@ -40,13 +55,18 @@ class UptechGrowthBookWrapper {
       _attributes.addAll(attributes);
     }
     _client = await _createLiveClient(
-        apiHost: apiHost, clientKey: clientKey, seeds: seeds);
+      apiHost: apiHost,
+      clientKey: clientKey,
+      features: finalFeatures,
+    );
     await refresh();
   }
 
   /// Initialize for use in automated test suite
   Future<void> initForTests(
-      {Map<String, dynamic>? seeds,
+      {Map<String, GBFeature> features = const <String, GBFeature>{},
+      @Deprecated('Instead use "features" as Map<String, GBFeature>')
+          Map<String, dynamic> seeds = const <String, dynamic>{},
```

Comment:
Why does `initForTests` not need the combining `finalFeatures` and `seeds` functionality that is being done in `init`? It looks like `_createTestClient` gets the `seeds` and `features` is never used. Is that correct?